doc/release-notes/iceoryx2-unreleased.md

```diff
@@ -9,6 +9,7 @@
  <!-- NOTE: Add new entries sorted by issue number to minimize the possibility of conflicts when merging. -->
 
  * Subscriber buffer size can be reduced [#19](https://github.com/eclipse-iceoryx/iceoryx2/issues/19)
+ * Introduce `IceAtomic` that supports up to 128bit atomics on 32-bit architecture with a ReadWriteLock [#200](https://github.com/eclipse-iceoryx/iceoryx2/issues/200)
```

Comment:
Philosophical question, `Ice` or `Iox`? I think you already used `Iox` in some other place

Reply:
Mmmhhh... `iox` is more consistent with the current philosophy but `ice` sounds literally cooler.

I would prefer `ice` in the future. Maybe also for the c binding, that every function starts with `ice`.

What do you think?!

Reply:
I'm leaning more towards `iox` but maybe just because I'm more used to it

Reply:
I renamed it into `IoxAtomic`